src/ranged.cpp

```diff
@@ -815,6 +815,10 @@ int Character::fire_gun( const tripoint &target, int shots, item &gun )
         const vehicle *in_veh = has_effect( effect_on_roof ) ? veh_pointer_or_null( here.veh_at(
                                     pos() ) ) : nullptr;
 
+        // Add gunshot noise if player is the one who shot
+        const item::sound_data gunnoise = gun.gun_noise( shots > 1 );
+        add_msg_if_player( _( "You shoot. %s" ), gunnoise.sound );
```

Comment:
```suggestion
        // Add gunshot noise if player is the one who shot
        const item::sound_data gunnoise = gun.gun_noise( shots > 1 );
        Creature *target_critter = get_creature_tracker().creature_at( aim );
        const std::string target_str = !!target_critter ? target_critter->disp_name() :
                                       get_map().disp_name( aim );
        add_msg_if_player( _( "You shoot %1$s with your %2$s. %3$s" ), target_str, gun.tname(),
                           uppercase_first_letter( gunnoise.sound ) );
```
This is just an optional suggestion. Just adds a few more details to the message:

![shoot_msg](https://user-images.githubusercontent.com/12537966/168203970-5f64534e-65de-4b26-a925-1ff5127534a6.png)

Reply:
Oh yeah, thats nicer

Reply:
You can also just use `at` in the message:
```
"You shoot at %1$s with your %2$s. %3$s"
```

Reply:
What if youre aiming at nothing though?

Reply:
```
get_map().disp_name( aim );
```
This part uses the name for the terrain or furniture at the location if there's no creature there

Reply:
Well, it seems like it's getting a little too complicated at that point, more code and longer lines to read...  I think its good to keep it simple myself